packages/vsx-registry/README.md

```diff
@@ -0,0 +1,4 @@
+# Theia - VSCode Extensions
```

Comment:
align with @vince-fugnitto template

document environment variable to configure vsx registry URL.